src/Resources/views/tailwind_2/_table.html.twig

```diff
@@ -141,7 +151,7 @@
                     </th>
                 {% endfor %}
                 {% if table.actions|length %}
-                    <th></th>
+                    <th class="w-20"></th>
```

Comment:
This should have a text content then, like "Aktions", so screenreaders can identify this row. The text can also be visually hidden/screenreader only if necessary. But the cell shouldn't be empty.